hooks/pre_start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+OPEN_FILES_RECOMMENDED=24576
+OPEN_FILES_LIMIT=$(ulimit -n)
+
+if [ "$OPEN_FILES_RECOMMENDED" -gt "$OPEN_FILES_LIMIT" ]; then
+    echo "WARNING: ulimit -n is $OPEN_FILES_LIMIT; $OPEN_FILES_RECOMMENDED is the recommended minimum."
+    echo "Try 'ulimit -n $OPEN_FILES_RECOMMENDED' before starting the node."
```

Comment:
After having run `bin/epoch start` and got this warning (and the node actually had started), I ran `ulimit -n 24576` as recommended then I re-ran `bin/epoch start` as the message would suggest. And this second invocation did not raise warning and returned exit status zero. Though the OS PID is the same! (I checked with `pgrep beam`).
So it looks to me that user experience is not optimal: user thinks that actually started node with correct ulimit though he/she did not .

Reply:
I suggest amending the message slightly in order not to induce to user to think that, after getting warning, simply running `ulimit -n` and then run `bin/epoch start` again and not getting warning fixed it. I suggest the following:
```
echo "WARNING: ulimit -n is $OPEN_FILES_LIMIT; $OPEN_FILES_RECOMMENDED is the recommended minimum."
echo "You are recommended to ensure the node is stopped and raise the maximum number of open files (try 'ulimit -n $OPEN_FILES_RECOMMENDED') before starting the node."
```

Reply:
Unfortunately this the best we can do for now without providing custom start script (and trashing rebar's one).